test_scripts/WebEngine/WebEngineProjection/007_Registration_reject_WEB_VIEW_is_absent_in_PT.lua

```diff
@@ -0,0 +1,56 @@
+--  Requirement summary:
+--  [RegisterAppInterface] REJECT
+--  [RegisterAppInterface] RegisterAppInterface and HMILevel
+--
+--  Description:
+--  Check that App will be rejected with HMI type WEB_VIEW 
+--  when WEB_VIEW AppHmiType is absent in application policies
+--
+--  Used precondition
+--  SDL, HMI are running on system.
+--  Mobile device is connected to SDL.
+--
+--  Performed steps
+--  appID->RegisterAppInterface(params)
+--
+--  Expected behavior:
+--     Policy table not contain WEB_VIEW.
+--     SDL->appID: REJECT, success:"false":RegisterAppInterface()
+---------------------------------------------------------------------------------------------------
+-- [[ Required Shared Libraries ]]
+local common = require('test_scripts/WebEngine/commonWebEngine')
+
+--[[ Local Variables ]]
+local appSessionId = 1
+local appHMITypeWebView = "WEB_VIEW"
+local appHMITypeMedia = "MEDIA"
+
+--[[ Local Functions ]]
+local function registerApp()
+  local session = common.createSession(appSessionId)
+  session:StartService(7)
+  :Do(function()
+      local corId = session:SendRPC("RegisterAppInterface", common.getParams(appSessionId))
+      EXPECT_HMINOTIFICATION("BasicCommunication.OnAppRegistered"):Times(0)
+      session:ExpectResponse(corId, { success = false, resultCode = "REJECTED" })
+  end)
+end
+
+--[[ General configuration parameters ]]
+config.application1.registerAppInterfaceParams.appHMIType = { appHMITypeWebView }
+config.application1.registerAppInterfaceParams.syncMsgVersion.majorVersion = 6
+config.application1.registerAppInterfaceParams.syncMsgVersion.minorVersion = 2
+
+--[[ Scenario ]]
+common.Title("Preconditions")
+common.Step("Clean environment", common.preconditions)
+common.Step("Update appHMIType in preloaded policy table", common.updatePreloadedPT,
+    { appSessionId, appHMITypeMedia })
```

Comment:
```suggestion
  { appSessionId, appHMITypeMedia })
```
Please remove redundant indents

Reply:
@VjKlepikov [done](https://github.com/LuxoftSDL/sdl_atf_test_scripts/pull/4/commits/e1b8a7c00153273c0320dc9b9b19fe8f2c9a6410#diff-82aaf8dae27f1b695f9cc2b4592ceba3R48)